Update Vercel integration to include setup instructions (#2554)
DataDog/integrations-extras: vercel/README.md

## Setup

### Datadog

To connect Datadog with Vercel, enable the integration and selct an API Key.

> 1. Open the Vercel Integration tile.

Reviewer: ```suggestion
1. Open the Vercel integration tile.
```

2. Select the **Configure** tab and select **Configure the Vercel integration**.

> 3. On the **Your Datadog account info** form, select **Select an API Key** and either choose an existing API Key or select **+ Create New** to use a new API Key for the Vercel Integration. Once your selection is made, select **Use API Key**. This will copy the selected API Key to your device's clipboard.

Reviewer: ```suggestion
3. On the **Your Datadog account info** form, select **Select an API Key** and either choose an existing API key or select **+ Create New** to use a new API key for the Vercel integration. 

4. After your selection is made, select **Use API Key**. This copies the selected API key to your device's clipboard.
```

> 4. Once the API Key is selected, you will be redirected back to the **Your Datadog account info** form. Select **Vercel > Add Integration** to complete the integraiton setup on Vercel.

Reviewer: ```suggestion
5. After the API key is selected, you are redirected back to the **Your Datadog account info** form. Select **Vercel > Add Integration** to complete the integration setup for Vercel.
```

### Vercel

> 1. The **Vercel > Add Integration** link will redirect you to Vercel's Datadog integration page.

Reviewer: ```suggestion
1. Click the **Vercel > Add Integration** link to be redirected to Vercel's Datadog integration page.
```

2. Select **Connect Account** to open the **Connect Datadog Account** form.

> 3. Select which Vercel Team to connect the integration to. (Teams that already have the integration installed will have an icon next to them labeled **Installed**)

Reviewer: ```suggestion
3. Select which Vercel Team to connect the integration to. (Teams that already have the integration installed have an icon next to them labeled **Installed**)
```

> 4. Select the radio button assocaited with either enabling the integration for all projects or a specific project that the team owns and then select **Connect Account** to save your changes.

Reviewer: ```suggestion
4. Select the radio button associated with either enabling the integration for all projects or a specific project that the team owns. Select **Connect Account** to save your changes.
```

> 5. Paste the API Key that was copied to your clipboard in Step 3 of the Datadog setup above into the box labeled **Your Datadog API Key**.

Reviewer: ```suggestion
5. Paste the API key that was copied to your clipboard in Step 4 of the Datadog setup above into the box labeled **Your Datadog API Key**.
```

## Troubleshooting

> If using the Vercel OTel Collector, the `serviceName` specified in `registerOTel` block of your [initializer][12] must match the Vercel Project name so that traces appear in the DataDog App with the appropriate logs and metrics.

Reviewer: ```suggestion
If you are using the Vercel OpenTelemetry Collector, the `serviceName` specified in `registerOTel` block of your [initializer][12] must match the Vercel Project name. This enables traces to appear in Datadog with the appropriate logs and metrics.
```
